README.md

```diff
@@ -1 +1,9 @@
-# reims-js-2103-project3-cash-is-wine-back
\ No newline at end of file
+# reims-js-2103-project3-cash-is-wine-back
+
+Project Cash is Wine
+
+The point is to create a online service of buyback and sale of ready-to-drink wines from individuals.
+
+Owner : Edgar Grantot
+
+Team : Allan Caplain, Bastien Le Brun, Marjorie Thomassin, Pascal Joly, Timothée Augustin
```

Comment:
good start :)

tip: put your names into links to your github profiles + tell more about the context : technologies, how much time for the project, how to clone and execute the code...